ash/src/vk/extensions.rs

```diff
@@ -17672,6 +17672,20 @@ impl ExtPipelinePropertiesFn {
     pub const SPEC_VERSION: u32 = 1u32;
 }
 #[allow(non_camel_case_types)]
+#[doc = "Implemented for all types that can be passed as argument to `p_pipeline_properties` in [`PFN_vkGetPipelinePropertiesEXT`]"]
+pub unsafe trait PFN_vkGetPipelinePropertiesEXT_param_p_pipeline_properties_validstructs {
```

Comment:
Better trait naming suggestions welcome!

Reply:
Camel-casing the argument name alone, maybe? Some risk of collisions, but not the worst breakage if we do hit it.

Reply:
I'd strip `p_` as unidiomatic in Rust as well. I think it's actually very descriptive in context, but the collision risk is unfortunate. Maybe `GetPipelinePropertiesPipelineProperties`? Reads odd, but makes sense with a moment's thought.

Reply:
How about `{}Param{}`? A delimiter makes sense to me, but "valid" feels redundant; implementing a trait is already intrinsically asserting a predicate.

Reply:
Good call; I wanted to convey that these are the `validstructs` for the parameter but that's already implicit when the parameter accepts implementers of this `trait`.